pkg/client/client.go

```diff
@@ -950,7 +990,7 @@ func (c *Client) MigrateCore(ctx context.Context) error {
 		}
 	}()
 
-	if err := m.UpgradeProvider(latestVersion); err != nil && err != migrate.ErrNoChange {
+	if err := m.UpgradeProvider("latest"); err != nil && err != migrate.ErrNoChange {
```

Comment:
I feel like this constant was defined and again undefined?

Reply:
yes, `latestVersion` is a wrong name for this type of constant, it's not pointing to the latest version number but signaling the UpgradeProvider to call m.m.Up().

Reply:
Fixed, constant moved to SDK